src/main/java/Event.java

```diff
@@ -0,0 +1,35 @@
+public class Event extends Task {
+
+    /**
+     * Main Constructor.
+     * @param description Description of given task.
+     */
+    Event(String description) {
+        super(description);
+    }
+
+    /**
+     * Overriden constructor to account for deadline and events that have timestamps.
```

Comment:
Not too sure if "Overriden" is the appropriate term here. I think "Overloaded constructor" may be more appropriate.